tests/ut/ir/transforms/test_inject_gm_pipe_buffer.py

```
@@ -36,6 +36,175 @@ def _run_inject(program: ir.Program) -> ir.Program:
+            updated = self.cube_consumer(a, out)
+            self.vector_producer(a, out)
+            return updated
+
+        @pl.function(type=pl.FunctionType.Orchestration)
+        def main(
+            self,
+            a: pl.Tensor[[16, 16], pl.FP16],
+            out: pl.Out[pl.Tensor[[16, 16], pl.FP16]],
+        ) -> pl.Tensor[[16, 16], pl.FP16]:
+            with pl.manual_scope():
+                updated, tid = pl.submit(self.group_func, a, out)
```

Ruff (0.15.14)

[warning] Unpacked variable `tid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)